src/server/routes/search.js

```diff
@@ -66,6 +74,11 @@ router.get('/autocomplete', (req, res) => {
 	handler.sendPromiseResult(res, searchPromise);
 });
 
+/**
+ * Regenerates search index. Only accessible to authenticated users
```

Comment:
Instead of "only accessible to authenticated users", maybe "restricted to administrators"? Not just any authed user can do it.